microsoft-edge/webview2/release-notes/index.md

```diff
@@ -120,6 +123,299 @@ description
 <!-- end of template -->
 
 
+<!-- ====================================================================== -->
+## 1.0.2849.39
+
+Release Date: October 21, 2024
+
+[NuGet package for WebView2 SDK 1.0.2849.39](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2849.39)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 130.0.2849.39 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+The following APIs have been promoted to Stable and are now included in this Release SDK.
+
+<!-- ---------- -->
+###### Configure the security warning when saving a file (`SaveFileSecurityCheckStarting` event)
+
+<!-- within Oct Release SDK -->
+
+Added a new `SaveFileSecurityCheckStarting` event.  Your app can register a handler on this event to get the file path, filename extension, and document origin URI information.  You can then apply your own rules to do actions such as the following:
+   * Allow saving the file without presenting a default security-warning UI about the file-type policy.
+   * Cancel the saving.
+   * Create your own UI to manage runtime file-type policies.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2` Class:<!-- todo: add CoreWebView2.GetDeferral Property ? -->
```

Comment:
add CoreWebView2.GetDeferral Property in .NET tab like other tabs?  if not, add comment about deliberately omitting
@victorhuangwq

Reply:
Yes we should have a getdeferral method for .NET

Reply:
Wait @peiche-jessica does CoreWebView2 have a GetDeferrel method?

Reply:
provisional resulting comment added:
```
##### [.NET/C#](#tab/dotnetcsharp)

<!--
.NET: CoreWebView2SaveFileSecurityCheckStartingEventArgs.GetDeferral Method
WinRT: CoreWebView2.GetDeferral Property
Win32: ICoreWebView2SaveFileSecurityCheckStartingEventArgs::GetDeferral
-->
```

Reply:
The CoreWebView2 entry should be for other events like `SaveFileSecurityCheckStarting` or `ScreenCaptureStarting`. Let me check why those entries are not generated for .NET and WinRT.

Reply:
Oh because they were promoted and they already existed.

Reply:
not clear what to do.  not clear if .md needs changes.  Closing Outdated comment, probably enter fresh